xml/art_migration.xml

```diff
@@ -0,0 +1,651 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<article version="5.1" xml:lang="en" xml:id="art.hos.migration"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Migration Guide</title>
+  <date><?dbtimestamp format="B d, Y"?></date>
+  <!-- <xi:include href="common_copyright_gfdl.xml"/> -->
+  <xi:include href="authors.xml"/>
+ </info>
+<sect1>
+  <title>Migration from HOS 5.x to HOS 5.SCP</title>
+  <para>
+    The migration procedure is based on the following assumptions.
+  </para>
+  <itemizedlist>
+    <listitem>
+      <para>
+	Customer is responsible for backing up all the control plane and deployer nodes to an external data store before starting migration
+      </para>
+      <itemizedlist>
+	<listitem>
+	  <para>
+	    We do not support rollback of migration process once started
+	   </para>
+	  </listitem>
+      </itemizedlist>
+    </listitem>
+    <listitem>
+      <para>
+	MML data will not be migrated (HOS 5.SCP includes a new MML backend
+	which will be installed fresh)
+    </para>
+    </listitem>
+    <listitem>
+      <para>
+	Any data not kept on a separate disk from the primary root partition will be lost during reimaging:
+    </para>
+    <itemizedlist>
+      <listitem>
+	<para>
+	  Swift objects
+	 </para>
+	</listitem>
+	<listitem>
+	  <para>
+	    Audit logs
+	   </para>
+	  </listitem>
+    </itemizedlist>
+    </listitem>
+    <listitem>
+      <para>
+	All computes are running supported Linux distribution:
+    </para>
+    <itemizedlist>
+      <listitem>
+	<para>
+	  SLES12 SP3 for SLES computes
+	 </para>
+	</listitem>
+	<listitem>
+	  <para>
+	    RHEL 7.3 for RHEL computes
+	   </para>
+	  </listitem>
+    </itemizedlist>
+    </listitem>
+    <listitem>
+      <para>
+	Only supported input model services
+    </para>
+    <itemizedlist>
+      <listitem>
+	<para>
+	  No embedded HOS Ceph or VSA
+	 </para>
+	</listitem>
+    </itemizedlist>
+    </listitem>
+    <listitem>
+      <para>
+	Only supported third-party integrations/extensions
+    </para>
+    <itemizedlist>
+      <listitem>
+	<para>
+	  No unsupported customized virtual environments
+	 </para>
+	</listitem>
+    </itemizedlist>
+    </listitem>
+  </itemizedlist>
+  <procedure>
+    <title>Preparation Phase</title>
+<step>
+  <para>
+    Run validation playbook to ensure that current cloud load and configuration
+    is compatible with migration process:
+  </para>
+  <screen>    (old-deployer)$ git clone -b hp/prerelease/newton https://gerrit.suse.provo.cloud/hp/hlm-migration-tools
```

Comment:
The prompts in the <screen/> elements do not match the SUSE style (we don't have a '$' as prompt).